client/src/Components/Navbar.jsx

```diff
@@ -0,0 +1,162 @@
+            </Link>
+            <Link to={`/${i18.language}`}>
+              <h3 className='text-white'> ڈی اردو </h3>
+            </Link>
+          </div>
+            <div className="flex gap-1">
+               {/* Globe button with onClick to toggle dropdown */}
+              <button 
+                className='px-2 py-0.5 m-1 rounded-lg text-white relative'
+                onClick={() => setDropdownVisible(!dropdownVisible)}
+              >
+                <FontAwesomeIcon icon={faGlobe} className='mr-1' />
+                {i18.language.toUpperCase()}
+              </button>
+
+              {/* Dropdown menu */}
+              {dropdownVisible && (
+                <div className='absolute mt-2 bg-gray-200 rounded-md shadow-lg'>
+                  <button 
+                    className='block px-4 py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('en')}
+                  >
+                    English (EN)
+                  </button>
+                  <button 
+                    className='block px-4 py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('de')}
+                  >
+                    German (De)
+                  </button>
+                  <button 
+                    className='block px-[25px] py-2 text-black hover:bg-gray-300'
+                    onClick={() => handleLanguageChange('ur')}
+                  >
+                    Urdu (UR)
+                  </button>
+                </div>
+              )}
+              {/* <Link to={`${getLanguagePath()}/login`}>
+                <button className='px-2 py-0.5 m-1 rounded-lg text-white'>{login}</button>
+              </Link>
+              <Link to={`${getLanguagePath()}/register`}>
+                <button className='px-2 py-0.5 m-1 rounded-lg text-white'>{register}</button>
+              </Link> */}
+              {isAuthenticated
+                ?
+                  <>
+                    <img src={user.picture} alt={user.name} className='rounded-full w-10 p-1'/>
+                    <button className='px-2 py-0.5 m-1 rounded-lg text-white' 
+                      onClick={() => logout({ logoutParams: { returnTo: window.location.origin },clientId: "NyjTsxGX6tAxOxKL6JkjzR6hY7MKhHn4" })}>
+                        {logOut}
+                    </button>
+                  </>
+                :
+                  <>
+                    <button className='px-2 py-0.5 m-1 rounded-lg text-white' 
+                      onClick={() => loginWithRedirect()}>
+                        {register}
+                    </button>
+                  </>
+              }
+            </div>
+        </div>
+      </div>
+
+        {/* This is 2nd Navbar */}
+        <div className={`flex bg-gray-700 ${i18.language === 'ur' ? 'gap-[775px]' : 'gap-[880px]'}
+                                            ${i18.language === 'de' ? 'gap-[700px]' : 'gap-[880px]'}`}>
```

Comment:
same comment as above how can the developer know how many pixels are needed here, comparing `775px` to `700px`

Reply:
The developer could evaluate that thing as per device, like for `large screens` (laptops etc...) it is good but for other screens it could be customize.